src/glass/src/app/pages/install-config-page/install-config-page.component.scss

```diff
@@ -0,0 +1,55 @@
+@import 'styles.scss';
+
+.glass-install-config-page {
+  height: 100%;
+  background-color: $glass-color-primary;
+  background-image: url('./../../../assets/images/jeremy-bishop-1braZySlEKA-unsplash.jpg');
+  background-blend-mode: multiply;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+
+  .mat-card {
```

Comment:
The `Next` button should be colored using the primary color like it is done for all buttons that guide the user through the installer.

```
Index: src/glass/src/app/pages/install-config-page/install-config-page.component.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/glass/src/app/pages/install-config-page/install-config-page.component.scss b/src/glass/src/app/pages/install-config-page/install-config-page.component.scss
--- a/src/glass/src/app/pages/install-config-page/install-config-page.component.scss	(revision 0906617794bc669f1771f048e81406451b75c698)
+++ b/src/glass/src/app/pages/install-config-page/install-config-page.component.scss	(date 1617004304085)
@@ -9,7 +9,7 @@
   background-repeat: no-repeat;
   background-position: center center;
 
-  .mat-card {
+  ::ng-deep .mat-card {
     width: 50%;
     @extend .glass-color-theme-white;
 
@@ -17,7 +17,7 @@
       margin-left: unset;
     }
 
-    ::ng-deep .mat-card.selected {
+    .mat-card.selected {
       border: 1px solid $glass-color-accent;
       color: $glass-color-white;
       background-color: rgba($glass-color-accent, 0.1);
@@ -33,7 +33,7 @@
       }
     }
 
-    ::ng-deep .mat-card {
+    .mat-card {
       border: 1px solid transparent;
       color: $eos-bc-gray-300;
 
@@ -51,5 +51,9 @@
         }
       }
     }
+
+    .mat-button {
+      @extend .glass-color-theme-primary;
+    }
   }
 }
```